### DISCUSSION TITLE
Edge to edge don't work in android

### DISCUSSION THREAD

**Discussion by salejo28** [jpudysz/react-native-unistyles#229] [Q&A]
![image](https://github.com/jpudysz/react-native-unistyles/assets/63069178/2d41c4d5-24c7-4ef5-93cb-d4e95799ae84)

Tengo el siguiente error y es que estoy usando react native unistyles 2.8.0-rc.6 y el edge to edge en android no funciona

**Comment by jpudysz** [comment]
Please update to most recent version and check common issues with edge to edge: 

https://www.unistyl.es/reference/faq/#my-android-app-is-not-edgetoedge-or-insets-are-incorrect

**Reply by salejo28** [reply]
No estoy usando react-native-keyboard-controller ni tampoco react-native-avoid-softinput, pero si react-native-screens y react-navigation sin embargo no tengo la opción statusBarTranslucent y sigue sin funcionar, y ya actualize react-native-unistyles a la última versión, cabe aclarar que estoy usando expo

**Comment by jpudysz** [comment]
Also, keep in mind that you need to specify insets manually like in this example:

https://reactnativeunistyles.vercel.app/reference/edge-to-edge/

Regarding react native warning - its not taking in count the insets, I can create a PR to fix that 🙏 I just need to check how it's implemented.

**Reply by salejo28** [reply]
Claro mira intente en mi proyecto actual y creando uno nuevo con el cli de react native de la comunidad, solo tuve que instalar react-native-unistyles colocar un console.warn o console.error y lo ejecutas, ni siquiera sin configurar los temas ni nada, y con el tema de los insets es muy engorroso con el tema del KeyboardAvoidingView, ya que cuando se abre el teclado el input queda con un margin enorme entre el keyboard y el mismo por el inset

**Reply by jpudysz** [reply]
So what's the issue about? You said it doesn't work, but now it's about avoiding view. If you think it's Unistyles issue please create a repro, otherwise it's impossible to help.
